src/Components/test/testassets/TestContentPackage/WasmFormComponent.razor

```diff
@@ -0,0 +1,74 @@
+﻿@attribute [RenderModeInteractiveWebAssembly]
```

Comment:
Might be simpler to disable prerendering here, then you don't need the `OperatingSystem.IsBrowser()` checks below.